cli/azd/pkg/infra/provisioning/bicep/bicep_provider.go

```diff
@@ -181,6 +181,8 @@ func (p *BicepProvider) EnsureEnv(ctx context.Context) error {
 
 	if scope == azure.DeploymentScopeResourceGroup {
 		if p.env.Getenv(environment.ResourceGroupEnvVarName) == "" {
+			// Prompt Resource Group supports creating a new resource group
+			// And prompts for a location as part of creating a new resource group
```

Comment:
Just wondering, why not prompt for the location further up (line 169) when `locParam.Value.(string)` is ""?

I got a little confused reading the comment above as I didn't see any prompting logic:

```
		// Check if location is a parameter and if it is not set in the AZD env
		// If it is not set, we need to prompt for it.
```